docs/distribution/uploading.md

```diff
@@ -126,3 +126,11 @@ You can find links to specific releases to public destinations on the releases t
 [app-center-home]: https://appcenter.ms/apps
 [apple-register-single-device]: https://help.apple.com/developer-account/#/dev40df0d9fa
 [apple-register-multiple-devices]: https://help.apple.com/developer-account/#/devebd34abb1
+
+## Troubleshooting
+
+### I'm getting "404: Not Found" when I try to upload my binaries to AppCenter distribution
+
+This error can happen if a member of your company got its corporate e-mail deactivated. Not having the member inside of the AppCenter's org, but also if the member was the one who created the API tokens to publish to AppCenter, it might end-up with the token getting rejected.
```

Comment:
P.S. I originally wrote this feedback much closer to when you originally created this PR, but I must've clicked away or the wrong thing since I just stumbled back here and saw it was a "saved draft." Sorry for the discrepancy.

Reply:
hey, @King-of-Spades no problem :D ... Yes, you are correct in this part "got its corporate e-mail deactivated."... We had a co-worker that made the tokens using his Gmail account. Once he left the company the e-mail was deactivated and pushing binaries to AppCenter stopped working.

Would you have a recommendation for a sentence that would explain this situation better? I can change it this week and send the corrections to you if you have something.

Reply:
I'd recommend the following changes:
- The header should be talking about the email deactivation aspect, and the 404 error context should be in the body, because 404 errors have so many causes for things not being found. 
- Instead of "a member/corporate e-mail deactivated" it should more explicitly state something like "the (user or account) who created the tokens" or "configured the distribution" etc. 

Basically just making sure we have a really clear connection between cause & effect.

Reply:
I can also polish a bit directly once we're on the same page. (Since I don't precisely have a reproduction, it's tricky for me to accurately rewrite it at present.)